locales/base/translation.json

```diff
@@ -551,8 +551,8 @@
   "escrowPaymentNotificationTitle": "Invite to {{mobile}} can't be claimed",
   "sendAndInvite": "Send & Invite",
   "inviteRewardsBanner": {
-    "title": "Send crypto to invite a friend",
-    "body": "Once they set up their {{appName}} wallet and connect their number, you’ll each get an exclusive NFT. <0>Earn up to 3 NFTs</0>"
+    "title": "Invite friends and collect NFTs",
+    "body": "Once they set up their {{appName}} wallet and connect their number, you’ll each get an exclusive NFT."
```

Comment:
i could also remove the code that allows the link to the support article (the <0> stuff)..... but wanted to leave the door open in case we want to add t's & c's quickly